src/main/java/mate/academy/hibernate/relations/service/impl/MovieServiceImpl.java

```diff
@@ -1,16 +1,34 @@
 package mate.academy.hibernate.relations.service.impl;
 
+import mate.academy.hibernate.relations.dao.MovieDao;
 import mate.academy.hibernate.relations.model.Movie;
 import mate.academy.hibernate.relations.service.MovieService;
+import mate.academy.hibernate.relations.util.DataProcessingException;
 
 public class MovieServiceImpl implements MovieService {
+    private MovieDao movieDao;
```

Comment:
Consider marking the `movieDao` field as final if it is not supposed to be changed after the initialization in the constructor. This is a good practice to follow as it makes the code easier to reason about and ensures the field is only set once.